Makefile

```diff
@@ -137,9 +153,17 @@ deploy-spoke: deploy-spoke-operator apply-spoke-cr
 deploy-spoke-operator: install-olm munge-spoke-csv bootstrap-secret
 	$(OPERATOR_SDK) run packagemanifests deploy/klusterlet/olm-catalog/klusterlet/ --namespace open-cluster-management --version $(CSV_VERSION) --install-mode SingleNamespace=open-cluster-management --timeout=10m
 
+deploy-spoke-kind: deploy-spoke-operator-kind apply-spoke-cr-kind
+
+deploy-spoke-operator-kind: install-olm-kind munge-spoke-csv bootstrap-secret-kind
+	$(OPERATOR_SDK) run packagemanifests deploy/klusterlet/olm-catalog/klusterlet/ --namespace open-cluster-management --version $(CSV_VERSION) --install-mode SingleNamespace=open-cluster-management --timeout=10m
+
 apply-spoke-cr:
 	$(SED_CMD) -e "s,quay.io/open-cluster-management/registration,$(REGISTRATION_IMAGE)," -e "s,quay.io/open-cluster-management/work,$(WORK_IMAGE)," deploy/klusterlet/config/samples/operator_open-cluster-management_klusterlets.cr.yaml | $(KUBECTL) apply -f -
```

Comment:
we may not need  this sed command for now, since it is only used for e2e

Reply:
Thanks for the good catch. I will remove the newly added `apply-spoke-cr-kind` target and reference.